content/300-guides/200-deployment/301-edge/550-deploy-to-deno-deploy.mdx

```diff
@@ -158,81 +158,50 @@ If you use VS Code and see the error `An import path cannot end with a '.ts' ext
-   1. Under **Choose a Data Proxy region**, select a Data Proxy location that is geographically close to your database.
-   1. Click **Create project**.
-
-      If the connection to the database is successful, Prisma Data Platform creates your project and redirects your browser to the **Get started** page. On the **Get started** page, you can [optionally link your GitHub repository](/data-platform/classic-projects/platform/projects/create#optional-enable-schema-synchronization) to keep your Prisma schema up-to-date.<br /><br />
-      For a project in Prisma Data Platform, you need a linked Prisma schema if you plan to use [Data Browser](/data-platform/classic-projects/platform/data-browser) and [Query Console](/data-platform/classic-projects/platform/query-console).<br /><br />
-      Deno Deploy requires a GitHub repository and you create that in [Create a repository and push to GitHub](#8-create-a-repository-and-push-to-github).
-
-      ![Prisma Data Platform - Create project - Paste database connection string, select Data Proxy region, enable static IPs](../../../doc-images/data-platform/projects/create-01-paste-connection-string.png)
-
-1. Create a Data Proxy connection string.
-
-   1. On the **Get started** screen, click **Create a new connection string**.
-   1. Enter a name for the connection string and click **Create**.
-   1. Copy the `prisma://` connection string to your clipboard.
-
-      ![Prisma Data Platform - Create project - Paste database connection string, select Data Proxy region, enable static IPs](../../../doc-images/data-platform/projects/create-03-copy-data-proxy-connection-string.png)
-
-   1. Save the connection string in a password manager or another safe location.
-   1. Click **Skip and continue to Data Platform** to open the Data Browser for your project.<br /><br />
-      When you test your application later, you come back to the Data Browser to review the records that your application creates.
-
-### What's next
-
-Next in this guide, you use the connection string in your local `.env` file to connect to your database through the Data Proxy.
-
-## 6. Configure the Data Proxy connection string in your environment
+Setup an Accelerate connection string and proceed to the next step.
 
-With the Data Proxy connection string copied, you can replace the direct PostgreSQL connection string that you used to create the database schema in your `.env` file.
+## 6. Configure Prisma Accelerate in your environment
 
-Prisma Client for the Data Proxy does not read `.env` files by default, so you must also install `dotenv-cli` locally.
+With the Accelerate connection string copied, you can replace the direct PostgreSQL connection string that you used to create the database schema in your `.env` file.
 
 ### Steps
 
-1. Install `dotenv-cli`.
```

Comment:
Dotenv CLI not needed any more for Deno? It's still used below in line 219.

Reply:
I'm not sure of this, hence, I'm adding the installation back in.